syllabus.rst

```diff
@@ -0,0 +1,53 @@
+########
+Syllabus
+########
+
+****
+Data
+****
+
+* the structure of raw brain imaging data;
+* loading, manipulating, showing and saving brain images;
+* coordinate systems and transforms for brain images;
+* working with multiple time courses.
+
+********
+Concepts
+********
+
+* writing and running diagnostics;
+* time-course interpolation to allow for slice-wise acquisition of brain
+  volumes;
+* cost-functions and numerical optimization for image alignment;
+* spatial transformations for image alignment: translations, rotations, zooms
+  and shears;
+* individual variability of sulci and other brain structure; methods of
+  reducing variability by automated alignment and warping; remaining variation
+  and statistical analysis;
+* smoothing / blurring prior to statistical analysis; cost and benefit;
+* multiple regression for modeling the effect of the experiment on time
+  courses of brain activity;
+* specifying a model of neural activity; transformation of neural activity to
+  predicted FMRI signal using a hemodynamic response function; modeling the
+  hemodynamic response;
+* estimating multiple regression models; hypothesis testing on multiple
+  regression models; the General Linear Model as generalization of multiple
+  regression;
+* inference on maps of statistics; correction for multiple comparisons;
+  family-wise error; false discovery rate.
+
+****************
+Working practice
```

Comment:
Um - I have the feeling lately that reproducible gets too much emphasis.  The key thing is quality, where reproducibility follows.   Can you think of a section title that captures that?